workshops/dodge/README.md

```diff
@@ -2,54 +2,696 @@
+```html
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Dodge</title>
+</head>
+<body>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.4.12/p5.js"></script>
+  <script src="https://cdn.rawgit.com/molleindustria/p5.play/master/lib/p5.play.js"></script>
+  <script src="game.js"></script>
+</body>
+</html>
+```
 
-## FAQ
+While you have `index.html` open in Cloud9, click `Preview > Live Preview File (index.html)` to see what the game looks like so far. Right now it's just a blank page, but we'll be changing that soon enough :wink:.
 
-### What library is this using?
+## Part II. The External JS File
 
-We are using two libraries:
+### Set Up
 
-- The [p5.js](http://p5js.org/) library
-- The [p5.play](http://p5play.molleindustria.org/) library (a library for games
-  built on top of p5.js)
+p5.js works by automatically calling two special functions: `setup()` and `draw()` to create the visuals on your webpage. We'll be writing our own code in these functions, so that p5.js can then run our code.
+
+Open up `game.js` and add them to the file (type them out, don't copy and paste!).
+
+```js
+function setup() {
+}
+
+function draw() {
+}
+```
+
+If you refresh the preview in Cloud9, you'll notice nothing has changed. It's still blank. Everything in due time, don't worry.
+
+How these functions work is that `setup()` is run only once, upon page load. `draw()` on the other hand, is run repeatedly after `setup()` finishes, and in this way, provides the basis for any animation or interaction you see in your project.
+
+Let's add something for `setup()` to set up.
+
+```js
+function setup() {
+  createCanvas(250, 250);
+}
+```
+
+[`createCanvas()`](http://p5js.org/reference/#p5/createCanvas) is a function that takes two arguments (two numbers for width and height, respectively) and creates an HTML canvas element (i.e., the thing that will house your project) of that size.
+
+Now we'll add something for the `draw()` function to do as well. Let's start by having it paint the background a dark blue color every frame.
+
+```
+function draw() {
+  background(0, 0, 100);
+}
+```
+
+`background()` is a function that sets the appearance of the background. Here we're using three arguments to define the red, green, and blue (RGB) values of a color, which will be the background color of our canvas. You can use `background` in some more ways, though, which are detailed in documentation [here](https://p5js.org/reference/#/p5/background).
+
+I've chosen this dark blue, but you can experiment with the RGB values and choose your own background color.
+
+Now that we've finished setting up, it's time to add the most important thing to our game: you, the player.
```

Comment:
yea i'm gonna veto this "you" addition because i refer to "you" as "the user" throughout the workshop, and explicitly avoid conflating "the player," "you," and "the user."